engines/stark/ui/menu/settingsmenu.h

```diff
@@ -31,6 +31,35 @@ namespace Stark {
 
 class VisualImageXMG;
 
+/**
+ * Manager of test sound
+ */
+class SoundManager {
+public:
+	SoundManager();
+	~SoundManager() {}
+
```

Comment:
The original game plays the testing sound in a special looping way. When the mouse is pressing down, the music plays in a loop and when pressing up, the music will play its last round. This sound manager helps to achieve this.